tests/Oryx.BuildImage.Tests/VersionInformationTest.cs

```diff
@@ -5,6 +5,7 @@
 
 using System;
 using Microsoft.Oryx.BuildScriptGenerator.DotNetCore;
+using Microsoft.Oryx.BuildScriptGenerator.Node;
```

Comment:
?

Reply:
I've moved `NodeConstants` from test project to this namespace